flake.nix

```diff
@@ -2,18 +2,23 @@
   description = "Core Lightning (CLN): A specification compliant Lightning Network implementation in C";
 
   inputs = {
-    nixpkgs.url = "github:NixOS/nixpkgs/nixos-24.05";
+    nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
```

Comment:
You said in your description that you would upgrade to `25.05`. Is there a specific reason why we should use `unstable` instead?

Reply:
I just realised that the stable channels are really more for whole NixOS systems. Seeing as we are just building a few packages here it will not require any extra effort or cause issues to be on unstable. And it means that developers on the project can use the latest tools available. Consumers of the flake might decide to override the input anyway (I do in my deployment).

I've updated the description but if you really want to stick to the stable channel that's fine with me as well.